onmt/decoders/transformer.py

```diff
@@ -232,6 +275,9 @@ def forward(self, tgt, memory_bank, step=None, **kwargs):
         attns = {"std": attn}
         if self._copy:
             attns["copy"] = attn
+        if with_align:
+            attns["align"] = attn_aligns[self.alignment_layer]  # `(B, Q, K)`
+            # attns["align"] = torch.stack(attn_aligns, 0).mean(0)  # All avg
```

Comment:
Can't we make this an option to allow easier experimentation? Maybe pass a specific value to one of the options you introduced?